author/ieee/ref/document-attributes.adoc

```diff
@@ -0,0 +1,87 @@
+
+For an _introduction to Metanorma AsciiDoc document attributes_ and how Metanorma uses them, see link:/author/topics/document-format/meta-attributes/[the corresponding topic].
+====
+
+== Document information
+
+`docnumber`:: In IEEE terms: the designation number for the document. Do not include the initial "P".
+
+`draft`:: The draft number. Do not include the initial "D".
+
+`isbn-pdf`:: The ISBN of the PDF document.
+`isbn-print`:: The ISBN of the print document.
+
+`stdid-pdf`:: The final standard identifier of the PDF document.
+`stdid-print`:: The final standard identifier of the print document.
+
+`doctype`::
+Document type. Choices:
++
+--
+* `standard` (default): This document is a Standard
+* `recommended-practice`: This document is a Recommended Practice
+* `guide`: This document is a Guide
+--
+
+`docsubtype`::
+Document subtype. Choices:
++
+--
+* `trial-use`: Document published for a limited period of time.
+--
+
+`confirmed-date`:: The date on which the document was approved.
+`issued-date`:: The date on which the document draft was circulated.
+
+`:keywords`:: Comma-delimited list of keywords associated with the document.
+
+`:doctitle:`:: Document title (also can be given as the first line of the Asciidoctor markup, prefaced by `= `.
+Do not include the words "Draft (Guide/Stadnard/Recommended Practice) for", they are supplied automatically
+by Metanorma.
+
+== Document participants
+
+`society`:: The IEEE society responsible for the document. Do not include the "IEEE" prefix.
+`committee`:: The committee responsible for the document.
+`working-group`:: The working group responsible for the document.
+`balloting-group`:: The Standards Association balloting group responsible for the document.
+
+IEEE requires listing the full membership of the working group, balloting group, and
+standards board responsible for the document. The current membership of the balloting group and
+standards board is normally provided by IEEE during editing; if the information is not provided in
+the document, dummy values will be provided to match those in the IEEE templates.
+This information is provided as follows; all names are given inline as full names.
+
+`wg_chair`:: The working group chair
+`wg_vicechair`:: The working group vice-chair
+`wg_secretary`:: The working group secretary
+`wg_members`:: The working group members, semicolon-delimited
+
+`balloting_group_members`:: The balloting group members, semicolon-delimited
+`std_board_chair`:: The standards board chair
+`std_board_vicechair`:: The standards board vice-chair
+`std_board_pastchair`:: The standards board past chair
+`std_board_secretary`:: The standards board secretary
+`std_board_members`:: The standards board members, semicolon-delimited. Any emeritus members should be given with an asterisk at the end of their names
```

Comment:
@opoudjis we have long used the agreed conventions that all document attributes use the "hyphen", not "underscores".

Reply:
Done